_projects/2024/100453237/100453237.Rmd

```diff
@@ -0,0 +1,589 @@
+
+The Economist Data Team represents the data in an elegant and simple style. The article's main graph is accompanied by an interactive table providing information about the cities, including their population and active mobility rate.
+
+I chose this graph because it seems an original way of presenting information on three variables. Besides, I am also really interested in issues related to urban mobility and possible differences between regions. The full article is available at this [link](https://www.economist.com/interactive/2024-walkable-cities).
+
+### Original chart
+
+The original graphic is an interactive ternary graphic in which a distinctive colour represents the three types of mobility mentioned above. In addition, when hovering over each point, the city's name associated with that point is indicated. The graph contains commentary, so as you scroll down, it is divided into different continents, and the article comments on the most salient results.
+
+![](images/graph.png){width="538"}
+
+## Replication
+
+### Libraries
+
+The first step is to load the libraries that will be used.
+
+```{r, warning = FALSE}
+
+library(tidyverse)
+library(ggthemes)
+library(scales)
+library(magick)
+library(ggtern)
+library(patchwork)
+library(plotly)
+library(lemon)
+library(shiny)
+library(showtext)
+
+```
+
+### Data loading
+
+Obtaining the data was easy since it is free and available on the [Cities Moving](https://citiesmoving.com/visualizations/) website. This website presents all the data used for The ABC mobility study, including several interactive charts showing differences across countries and continents.
+
+The main issue with the dataset was that it is updated from time to time so it is impossible to find which are the cities and observations that were included in the original chart. The data contains a variable where you can check which are the last observations, but this does not mean that these were the observations initially included since there are new cities that were not included before. To mitigate this situation, I created a table from the information contained in The Economist's article. Below the graph, there is a table that contains the names of the 794 cities included in the study, but the names were not written in the same way that they are written in the mobility dataset. Therefore, I changed manually the names, erasing accents and modifying the special characters. This change provided me the possibility of joining the two tables, obtaining a final data of 773 observations, which was closer to 794 than the initial filter with the first table (866 observations).
+
+```{r}
+## Preparing the data frame
+
+mobility_data <- read_csv("D:/COMPUTATIONAL SOCIAL SCIENCE/First semester/Data visualization/data/ModalShare.csv")
+
+cities <- read_csv2("D:/COMPUTATIONAL SOCIAL SCIENCE/First semester/Data visualization/data/cities.csv")
```

Comment:
Data should be part of your PR. Put them close to the Rmd, and these lines should be just `read_csv("ModalShare.csv"`, etc.